src/Text/Pandoc/Readers/DokuWiki.hs

```diff
@@ -471,19 +479,24 @@ table = do
   let (headerRow, body) = if firstSeparator == '^'
                             then (head rows, tail rows)
                             else ([], rows)
-  let attrs = (AlignDefault, ColWidthDefault) <$ transpose rows
+  -- Since Pandoc only has column level alignment, we have to make an arbitrary
+  -- choice of how to reconcile potentially different alignments in the row.
+  -- Here we end up assuming that the alignment of the header / first row is
+  -- what the user wants to apply to the whole thing.
+  let attrs =  map (\(a, _) -> (a, ColWidthDefault)) . head $ rows
```

Comment:
`head` makes me nervous because it's partial, even though here `many1` guarantees a nonempty list.
Could we use `headMay` from Safe, or a pattern match, or some other non-partial function?